lib/array.rb

```diff
@@ -0,0 +1,11 @@
+class Array
+
+  def my_inject &block
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `block`. If it's necessary, use `_` or `_block` as an argument name to indicate that it won't be used. You can also write as `my_inject(*)` if you want the method to accept any arguments but don't care about them.